EIPS/eip-7870.md

```diff
@@ -64,7 +64,7 @@ Node operators typically run both an **Execution Layer (EL)** client and a **Con
   - Running a node involves frequent writes (e.g., database updates, logs). Ensure that the SSD’s Total Bytes Written (TBW) rating is sufficient for multi-year operation.
 - **Capacity Considerations**  
   - As of January 2025, 2 TB can still work, but daily chain growth makes 4 TB more future-proof.  
-  - While EIP-4444 aims to reduce historical storage requirements, the timeline for EIP-4444 remains uncertain.
+  - While [EIP-4444](./eip-4444.md) aims to reduce historical storage requirements, the timeline for [EIP-4444](./eip-4444.md) remains uncertain.
```

Comment:
As you see in the linter, one cannot mention a draft EIP in an EIP that goes to Review. 
```
proposal `eip-4444.md` is not stable enough for a `status` of `Review`
```